charts/ocis/values.yaml

```diff
@@ -1020,6 +1020,40 @@ features:
         # - role_name: user-light
         #   claim_value: ocisGuest
         mapping: []
+
+      # Configure OIDC space assignment. If activate, oCIS will read the space assignements from the oidc token
+      spaceAssignment:
+        # @schema
+        # type: boolean
+        # required: true
+        # @schema
+        # -- enable OIDC space assignment.
+        enabled: false
+        # @schema
+        # type: string
+        # required: true
+        # @schema
+        # -- The name of the OIDC claim holding the space assignment
+        claim: claimname
```

Comment:
In my opinion, we should default to "" (empty string here). This is at least how we do it for all other cases.